onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -765,6 +765,24 @@ def test_session_with_ortvalue_input(ortvalue):
             # The constructed OrtValue should still be valid after being used in a session
             self.assertTrue(np.array_equal(ortvalue2.numpy(), numpy_arr_input))
 
+    def testRunModelWithCudaCopyStream(self):
+        available_providers = onnxrt.get_available_providers()
+
+        if (not 'CUDAExecutionProvider' in available_providers):
+            print("Skipping testRunModelWithCudaCopyStream when CUDA is not available")
+        else:
+            # adapted from issue #4829 for a race condition when copy is not on default stream
+            # note:
+            # 1. if there are intermittent failure in this test, something is wrong
+            # 2. it's easier to repro on slower GPU (like M60, Geforce 1070)
+
+            # to repro #4829, uncomment the line below to run copy in a separate stream
+            onnxrt.capi._pybind_state.set_do_copy_in_default_stream(False)
```

Comment:
>onnxrt.capi._pybind_state.set_do_copy_in_default_stream(False) [](start = 12, length = 62)

This is intended to test if CI would fail as in local tests.

Reply:
Confirmed this line would cause test failure, commenting out now.